data-serving/data-service/src/model/case-revision.ts

```diff
@@ -0,0 +1,24 @@
+import { CaseDocument, caseSchema } from './case';
+
+import { ObjectId } from 'mongodb';
+import mongoose from 'mongoose';
+
+const caseRevisionSchema = new mongoose.Schema({
+    case: {
+        type: caseSchema,
+        required: true,
+    },
+}).index(
+    { 'case._id': 1, 'case.revisionMetadata.revisionNumber': -1 },
```

Comment:
What are we using this index for? Is it simply to enforce uniqueness?

Reply:
This will be to speed up the query for fetching a particular revision or all revisions for a given document. Happy to hold off on it until we have that functionality if you rpefer